static/src/javascripts/projects/common/modules/support/banner.ts

```diff
@@ -126,10 +127,12 @@ const buildBannerPayload = async (): Promise<BannerPayload> => {
 		mvtId: getMvtValue() ?? 0,
 		countryCode: getCountryCode(),
 		weeklyArticleHistory: getWeeklyArticleHistory(storage.local),
+		articleCountToday: getArticleViewCountForDays(1) as number,
```

Comment:
Have you added the type assertion because `history.js` is untyped?

Reply:
correct